Automattic/wp-calypso
# Nav Unification: Announcement Modal

#### Changes proposed in this Pull Request

* Introduces the customer modal giving the users context on the nav unification changes.

#### Testing instructions

* Visit the [Calypso.live](https://calypso.live/?branch=nav-unification/customer-modal) link.
* You should be presented with the Nav Unification modal for the first time.
* After dismissing the modal you should not see this modal again.

Fixes https://github.com/Automattic/wp-calypso/issues/47049
